Resources/Private/Scripts/ColorPickerEditor/src/NeosColorPicker/ColorPicker.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import reactCSS from 'reactcss'
+import merge from 'lodash/merge'
+
+import { ColorWrap, Saturation, Checkboard } from 'react-color/lib/components/common'
+import Alpha from './Alpha'
+import Hue from './Hue'
+import Fields from './Fields'
+import PresetColors from './PresentColors'
+import {IconButton} from '@neos-project/react-ui-components';
+
+export const ColorPicker = ({ width, rgb, hex, hsv, hsl, onChange, onReset, onSwatchHover,
+  disableAlpha, presetColors, renderers, styles: passedStyles = {}, className = '' }) => {
```

Comment:
Is `disableAlpha` forwarded to the fields? I cannot see where and PHPStorm shows it's unused.

Reply:
Maybe we can make this a future feature. Until this I removed the variable.